app/assets/javascripts/authoring/awareness.js

```diff
@@ -1320,7 +1321,7 @@ var updateStatus = function(flash_team_in_progress){
             type: 'post',
             data: {"localStatusJSON": localStatusJSON, "authenticity_token": authenticity_token}
         }).done(function(data){});
-    }, 2000);
+    });
```

Comment:
With this timer milliseconds removed, we may no longer need to create a `timer` above and set/reset it. Perhaps remove that?